### DISCUSSION TITLE
Only streaming the selected game window.

### DISCUSSION THREAD

**Discussion by robethang** [LizardByte/community#253] [Sunshine] [answered]
Hi.
Is there a way to only streaming the selected game window. and the computer screen can still be used for other things when minimizing the game window. 
Thank you

**Comment by jimmt41** [comment]
I've read a few msg here where I believe people solve this problem using a virtual monitor and having the application run on that monitor.

**Comment by yremac** [comment]
Have you found a way to solve your problem?

**Reply by robethang** [reply]
I'm still not

**Comment by ReenigneArcher** [comment, marked as answer]
Sunshine streams the entire desktop, not specific windows.
